src/components/MoviesList/MoviesList.tsx

```diff
@@ -6,10 +6,12 @@ interface Props {
   movies: Movie[];
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList: React.FC<Props> = ({ movies }) => {
+  return (
+    <div className="movies">
+      {movies.map(movie => (
+        <MovieCard key={movie.imdbId} movie={movie} />
+      ))}
+    </div>
+  );
+};
```

Comment:
I think these changes are pointless

Reply:
Thank you. So it would be better to remove return because it's useless?